### PR TITLE
Add Amazon RDS us-east-2, GovCloud Root, and us-gov-west-1 certificates

### PR DESCRIPTION
**Pull request by jacqt**
Links to the added certificates:
http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.SSL.html

(US East Ohio)
https://s3.amazonaws.com/rds-downloads/rds-ca-2015-us-east-2.pem

(GovCloud 2017)
https://s3-us-gov-west-1.amazonaws.com/rds-downloads/rds-ca-bundle-us-gov-west-1.pem

### CODE REVIEW

File: lib/protocol/constants/ssl_profiles.js [hunk 1]
```diff
@@ -550,6 +550,102 @@ exports['Amazon RDS'] = {
...
+    + '9mlnSQQlbN/wqIIXbkrcgFcHoXG9Odm/bDtUwwwDaiEhXVfeQom3G77QHOWMTCGK\n'
+    + 'qadwuh5msrb17JdXZoXr4PYHDKP7j0ONfAyFNER2+uecblHfRSpVq5UeF3L6ZJb8\n'
+    + 'fSw/GtAV6an+/0r+Qm+PiI2H5XuZ4GmRJYnGMhqWhBYrY7p3jtVnKcsh39wgfUnW\n'
+    + 'AvZEZG/yhFyAZW0Essa39LiL5VSq14Y1DOj0wgnhSY/9WHxaAo1HB1T9OeZknYbD\n'
+    + 'fl/EGSZ0TEvZkENrXcPlVA==\n'
+    + '-----END CERTIFICATE-----\n',
+
+    /**
+     * Amazon RDS us-gov-west-1 CA certificate from 2017 to 2022
+     *   CN = Amazon RDS us-gov-west-1 CA
```
Comment (dougwilson) [inline lib/protocol/constants/ssl_profiles.js]: Looks like there are two certs but this block only describes one of them. Can you add a block for the other?

Comment (jacqt) [reply lib/protocol/constants/ssl_profiles.js]: Done :+1:

**Comment by jacqt** [comment]
Friendly bump :)

Separated out the block for AWS Gov as requested

**Comment by jacqt** [comment]
Just bumping this again @dougwilson

**Comment by wcgcoder** [comment]
@jacqt just dropping a note to say THANKS because this PR is the only reference to where to find the GovCloud RDS CA that I've found searching around.

**Pull request closed without merging**